docs/docs/testing-your-assistant.mdx

```diff
@@ -239,10 +264,10 @@ To evaluate entity extraction we apply a simple tag-based approach. We don't con
 entity type tags on a per token basis. For location entity like “near Alexanderplatz” we
 expect the labels `LOC LOC` instead of the BILOU-based `B-LOC L-LOC`. 
 
-Our approach is more lenient
-when it comes to evaluation, as it rewards partial extraction and does not punish the splitting of entities.
+Our approach is more lenient when it comes to evaluation, as it rewards
```

Comment:
General comment: I'm not sure that this section needs the discussion of BILOU tags. Philosophically, it seems like it should focus on what rasa does, rather than what it does not do.

Reply:
I do not have a hard opinion on this but it seems like important information for users to know. I am not sure if this is explained somewhere else in the docs. @tabergma Any opinions on this since you might have added this originally?

Reply:
I did not add it, but seem to be relevant information. We did not had any section on BILOU tags until I added a short section last week or so (https://github.com/RasaHQ/rasa/blob/master/docs/docs/nlu-training-data.mdx#bilou-entity-tagging). I also think we should keep this information here or we should add it to the section around BILOU tags. But would state it somewhere. If we mention BILOU here, we should for sure reference the new section.

Reply:
Ah, ok. I think what's confusing about this text is that it implies at 240/ 265 that we don't use BILOU tags, so it seemed weird to me to then talk extensively about them. Probably then it makes sense to reword the initial text so that it doesn't seem to imply that.

Reply:
Added a small rewording to indicate that we don't use the tags *exactly* . Also added a link to the section of bilou tags.